### DISCUSSION TITLE
Same atom, multiple interactions

### DISCUSSION THREAD

**Discussion by f-oliva** [chemosim-lab/ProLIF#91] [Q&A] [answered]
Hello everyone,
I was trying to use prolif to calculate the interactions between RNA and a ligand. Looking at the results I started to think that even in the bitvector (using bitvector_atoms) if an atom of the ligand is doing multiple interactions with RNA only one of them is reported. I sat up a toy system with formaldehyde manually placed to form 2 hydrogen bonds with the amino group of an adenine and only one of them is reported. Is there something that I'm missing?
Thank you

**Comment by cbouy** [comment, marked as answer]
Hi @Ciskio, 

You are right, for a given pair of residues (your ligand and a nucleotide in your case) and an interaction type (HBond, pi-stacking... etc.) ProLIF currently only returns the first occurrence it finds.
I have plans to allow returning all occurrences of an interaction for a given pair in the future instead of only the first one, but that will take some time and will probably have to wait for next year.
